python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -102,7 +105,7 @@ def fetchReferences(self, dataRef, exposure):
     def _makeArgumentParser(cls):
         parser = lsst.pipe.base.ArgumentParser(name=cls._DefaultName)
         parser.add_id_argument("--id", "deepCoadd_forced_src", help="data ID, with raw CCD keys + tract",
-                               ContainerClass=lsst.coadd.utils.CoaddDataIdContainer)
+                               ContainerClass=lsst.coadd.utils.ExistingCoaddDataIdContainer)
```

Comment:
I think it's fine to leave it as just CoaddDataIdContainer (I had asked you guys about this earlier and you leaned the other way, but I'm happy to revert!)